### PR TITLE
data release 2023-04-12

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

**Review by jehangiramjad — APPROVED** [review]
Based on a quick look, it looks expected, mostly?

File: deploy/storage/bigquery.version [hunk 1]
```diff
@@ -1 +1 @@
-datcom-store.dc_kg_2023_03_20_23_14_35
\ No newline at end of file
+datcom-store.dc_kg_2023_04_04_00_35_41
```
Comment (pradh) [inline deploy/storage/bigquery.version:1]: Was there a newer one - dc_kg_2023_04_10_23_19_57 - that completed last night?

https://groups.google.com/a/google.com/g/datacommons/c/INNVIJaDNWM

File: internal/server/v1/propertyvalues/golden/bulk_property_values_in/containedIn1.json [hunk 2]
```diff
@@ -22550,38 +22550,6 @@
...
-          "provenanceId": "dc/base/WIDAP_PowerPlants"
-        },
-        {
-          "name": "Gilroy Energy Center, LLC for King City",
-          "types": [
-            "PowerPlant"
-          ],
-          "dcid": "dc/yk2qzf8sq2j77",
-          "provenanceId": "dc/base/WIDAP_PowerPlants"
-        },
```
Comment (pradh) [inline internal/server/v1/propertyvalues/golden/bulk_property_values_in/containedIn1.json:22584]: Expected since we removed the import: cl/517516453

**Review by pradh — COMMENTED** [review]
Thanks!

File: deploy/storage/bigquery.version (hunk 1 above)
Comment (chejennifer) [reply deploy/storage/bigquery.version]: updated!

**Review by pradh — APPROVED** [review]
Thanks

**Pull request merged**